mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -404,16 +405,28 @@ public void sendMail(MailAddress sender, Collection<MailAddress> recipients, Mim
             .build());
     }
 
+    @Override
     public void sendMail(Mail mail) throws MessagingException {
         sendMail(mail, Mail.DEFAULT);
     }
 
+    @Override
+    public void sendMail(Mail mail, long delay, TimeUnit unit) throws MessagingException {
+        sendMail(mail, Mail.DEFAULT, delay, unit);
+    }
+
     @Override
     public void sendMail(Mail mail, String state) throws MessagingException {
         mail.setState(state);
         sentMails.add(fromMail(mail));
     }
 
+    @Override
+    public void sendMail(Mail mail, String state, long delay, TimeUnit unit) throws MessagingException {
+        mail.setState(state);
+        sentMails.add(fromMail(mail)); // FIXME delay ignored here for now
```

Comment:
I think it is OK for now, this is a testing class. Maybe you can add delay + unit as a SentMail property?